src/main/java/su/litvak/chromecast/api/v2/Channel.java

```diff
@@ -111,18 +111,27 @@ public void run() {
         @Override
         public void run() {
             while (!stop) {
+                CastChannel.CastMessage message = null;
                 try {
-                    CastChannel.CastMessage message = read();
+                    message = read();
                     if (message.getPayloadType() == CastChannel.CastMessage.PayloadType.STRING) {
-                        LOG.debug(" <-- {}", message.getPayloadUtf8());
+
                         final String jsonMSG = message.getPayloadUtf8().replaceFirst("\"type\"", "\"responseType\"");
+                        if (!jsonMSG.contains("responseType")) {
```

Comment:
there is a message that gets sent (I think from Google Music) that does not have a responseType value. I think it only had two values, 'event' and something else.